testdata/godoc-v3.x.txt

```diff
@@ -775,10 +775,10 @@ var VersionFlag Flag = &BoolFlag{
 type FlagBase[T any, C any, VC ValueCreator[T, C]] struct {
 	Name string // name of the flag
 
-	Category    string // category of the flag, if any
-	DefaultText string // default text of the flag for usage purposes
-	FilePath    string // file path to load value from
-	Usage       string // usage string for help output
+	Category    string   // category of the flag, if any
+	DefaultText string   // default text of the flag for usage purposes
+	FilePath    []string // file paths to load value from
```

Comment:
Should it be named to `FilePaths` then?

Reply:
Thats what I initially was about to do but thought to keep the name same since behavior is same. I can change it. @meatballhat any thoughts ?

Reply:
With many file paths, what will happen if first path does not exist? Will value from the second path overwrite the first?

Reply:
I'm in favor of a rename to `FilePaths` 👍🏼 and to @abitrolly's point it'd be nice to have clearer documentation around the expected behavior 🙇🏼 (in a later PR!)